ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -159,6 +179,8 @@
         -v {{ dockerInfo["DockerRootDir"] }}/containers/:/containers
         -v {{ docker_sock | default('/var/run/docker.sock') }}:/var/run/docker.sock
         -p {{ invoker.port + groups['invokers'].index(inventory_hostname) }}:8080
+        {% if jmx.enabled %} -p {{ jmx.basePortInvoker + groups['invokers'].index(inventory_hostname) }}:{{ jmx.basePortInvoker + groups['invokers'].index(inventory_hostname) }} {% endif %}
+        {% if jmx.enabled %} -p {{ jmx.rmiBasePortInvoker + groups['invokers'].index(inventory_hostname) }}:{{ jmx.rmiBasePortInvoker + groups['invokers'].index(inventory_hostname) }} {% endif %}
```

Comment:
Are we completely sure, that a user is not able to reach this port from an useraction?

Reply:
As long as we have network isolation between management and user container plain on the invokers we must be safe. Now they are separated into 2 subnets 172.x.x.x and 192.x.x.x afaik.